docs/docs/community/usage-tracking.mdx

```diff
@@ -0,0 +1,239 @@
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>import_mlflow</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>create_experiment</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>create_run</td>
+      <td>Whether the packages in [PACKAGES_TO_CHECK_IMPORT](https://github.com/mlflow/mlflow/blob/c71fd0d677c1806ba2d5928398435c4de2c25c0e/mlflow/telemetry/constant.py#L19) are imported or not</td>
+    </tr>
+    <tr>
+      <td>create_logged_model</td>
+      <td>Flavor of the model (e.g. langchain, sklearn)</td>
+    </tr>
+    <tr>
+      <td>create_registered_model</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>create_model_version</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>create_prompt</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>start_trace</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>log_assessment</td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>evaluate</td>
+      <td>None</td>
+    </tr>
+  </tbody>
+</Table>
+
+## Why is MLflow Telemetry Opt-Out?
+
+MLflow uses an opt-out telemetry model to help improve the platform for all users based on real-world usage patterns.
+Collecting anonymous usage data by default allows us to:
+
+- Understand how MLflow is being used across a wide range of environments and workflows
+- Identify common pain points and identify feature improvements area more effectively
+- Measure the impact of changes and ensure they improve the experience for the broader community
+
+If telemetry were opt-in, only a small, self-selected subset of users would be represented, leading to biased insights and potentially misaligned priorities.
+We are committed to transparency and user choice. Telemetry is clearly documented, anonymized, and can be easily disabled at any time through configuration.
+This approach helps us make MLflow better for everyone, while giving you full control. Check [`what we are doing with this data`](#what-are-we-doing-with-this-data) section for more information.
+
+## How to opt-out?
+
+MLflow supports opt-out telemetry through either of the following environment variables:
+
+- **MLFLOW_DISABLE_TELEMETRY=true**
+- **DO_NOT_TRACK=true**
+
+Setting either of these will **immediately disable telemetry**, no need to re-import MLflow or restart your session.
+
+:::note
+MLflow automatically disables telemetry in the following CI environments.
+If you’d like support for additional CI environments, please [open an issue on our GitHub repository](https://github.com/mlflow/mlflow/issues).
+
+- CI
+- Github Actions
+- CircleCI
+- GitLab CI
+- Jenkins
+- Travis
+- Azure Pipeline
+- BitBucket
+- CodeBuild
+- BuildKite
+  :::
+
+### Scope of the setting
+
+- The environment variable only takes effect in processes where it is explicitly set or inherited.
+- If you spawn subprocesses (e.g., via subprocess.run) from a clean environment, those subprocesses may not inherit your shell’s environment, and telemetry could remain enabled.
```

Comment:
https://github.com/mlflow/mlflow/pull/16917/files#r2238307595

Reply:
For showing when the environment variables are not inherited?

Reply:
Yep